IFC 4.3/SectionedSolidHorizontal_1/UT_SectionedSolidHorizontal_1_0.ifc

```diff
@@ -0,0 +1,195 @@
+#216= IFCARBITRARYCLOSEDPROFILEDEF(.AREA.,'SampleProfile',#217);
+#217= IFCPOLYLINE((#219,#221,#223,#225,#227,#229,#231,#233,#219));
+#219= IFCCARTESIANPOINT((300.,-600.));
+#221= IFCCARTESIANPOINT((300.,-150.));
+#223= IFCCARTESIANPOINT((1250.,-100.));
+#225= IFCCARTESIANPOINT((1250.,0.));
+#227= IFCCARTESIANPOINT((-1250.,0.));
+#229= IFCCARTESIANPOINT((-1250.,-100.));
+#231= IFCCARTESIANPOINT((-300.,-150.));
+#233= IFCCARTESIANPOINT((-300.,-600.));
+#235= IFCCARTESIANPOINT((300.,-600.));
+#237= IFCARBITRARYCLOSEDPROFILEDEF(.AREA.,'SampleProfile',#238);
+#238= IFCPOLYLINE((#240,#242,#244,#246,#248,#250,#252,#254,#240));
+#240= IFCCARTESIANPOINT((300.,-600.));
+#242= IFCCARTESIANPOINT((300.,-150.));
+#244= IFCCARTESIANPOINT((1250.,-100.));
+#246= IFCCARTESIANPOINT((1250.,0.));
+#248= IFCCARTESIANPOINT((-1250.,0.));
+#250= IFCCARTESIANPOINT((-1250.,-100.));
+#252= IFCCARTESIANPOINT((-300.,-150.));
+#254= IFCCARTESIANPOINT((-300.,-600.));
+#256= IFCCARTESIANPOINT((300.,-600.));
+#258= IFCARBITRARYCLOSEDPROFILEDEF(.AREA.,'SampleProfile',#259);
+#259= IFCPOLYLINE((#261,#263,#265,#267,#269,#271,#273,#275,#261));
+#261= IFCCARTESIANPOINT((300.,-800.));
+#263= IFCCARTESIANPOINT((300.,-150.));
+#265= IFCCARTESIANPOINT((1250.,-100.));
+#267= IFCCARTESIANPOINT((1250.,0.));
+#269= IFCCARTESIANPOINT((-1250.,0.));
+#271= IFCCARTESIANPOINT((-1250.,-100.));
+#273= IFCCARTESIANPOINT((-300.,-150.));
+#275= IFCCARTESIANPOINT((-300.,-800.));
+#277= IFCCARTESIANPOINT((300.,-800.));
+#279= IFCARBITRARYCLOSEDPROFILEDEF(.AREA.,'SampleProfile',#280);
+#280= IFCPOLYLINE((#282,#284,#286,#288,#290,#292,#294,#296,#282));
+#282= IFCCARTESIANPOINT((300.,-800.));
+#284= IFCCARTESIANPOINT((300.,-150.));
+#286= IFCCARTESIANPOINT((1250.,-100.));
+#288= IFCCARTESIANPOINT((1250.,0.));
+#290= IFCCARTESIANPOINT((-1250.,0.));
+#292= IFCCARTESIANPOINT((-1250.,-100.));
+#294= IFCCARTESIANPOINT((-300.,-150.));
+#296= IFCCARTESIANPOINT((-300.,-800.));
+#298= IFCCARTESIANPOINT((300.,-800.));
+#301= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(0.),$,$,$
+,#79);
+#302= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(2500.),$,
+$,$,#79);
+#303= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(5000.),$,
+$,$,#79);
+#304= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(17500.),$
+,$,$,#79);
+#305= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(20000.),$
+,$,$,#79);
+#306= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(25000.),$
+,$,$,#79);
+#307= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(27500.),$
+,$,$,#79);
+#308= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(40000.),$
+,$,$,#79);
+#309= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(42500.),$
+,$,$,#79);
+#310= IFCPOINTBYDISTANCEEXPRESSION(IFCNONNEGATIVELENGTHMEASURE(45000.),$
+,$,$,#79);
+#311= IFCSHAPEREPRESENTATION(#62,'Body','AdvancedSweptSolid',(#86));
+#319= IFCBEAM('0L0ZdYU4990hP0BIdOvRdh',#11,'MG',$,$,#74,#75,$,
```

Comment:
The beam is orphaned from the spatial structure.
A containment relationship is needed such as 
#2 = IFCRELCONTAINEDINSPATIALSTRUCTURE('3jCDKvZ0d7lKBGdztX1NY3', #3, $, $, (#319), #56);